src/controllers/profileController.ts

```diff
@@ -45,7 +51,42 @@ export default class ProfileController {
     return withUserFromRequest(req, async user =>
       withValidatedOrValidationError(
         EmailString.decode(req.body.work_email),
-        workEmail => this.profileService.updateProfile(user, workEmail)
+        async workEmail => {
+          const errorResponseOrSuccessResponse = await this.profileService.updateProfile(
+            user,
+            workEmail
+          );
+          log.info(
+            "errorResponseOrSuccessResponse: %s",
+            JSON.stringify(errorResponseOrSuccessResponse, null, 4)
+          );
+          log.info(
+            "typeof errorResponseOrSuccessResponse: %s",
+            typeof errorResponseOrSuccessResponse
+          );
+          return errorResponseOrSuccessResponse.map(response => {
+            const emailText = `Ciao ${user.givenName},
+la tua email di lavoro è stata modificata con successo, da questo momento riceverai le comunicazioni al nuovo indirizzo da te scelto.`;
+            // tslint:disable-next-line:no-floating-promises
+            this.emailService
+              .send({
+                from: getRequiredEnvVar("EMAIL_SENDER"),
```

Comment:
that would be better to pass this as a parameter to the emailservice (so all the configuration happens at top level) and use `getRequiredEnvVar` only into `app.ts`